Marlin/Marlin_main.cpp

```diff
@@ -8434,6 +8435,21 @@ inline void gcode_M211() {
   SERIAL_ECHOLNPAIR(" " MSG_Z, soft_endstop_max[Z_AXIS]);
 }
 
+/**
+ * M217: Send text to serial output
+ *
+ * Usage: M217 some text
+ * Text must be prefixed by "//" and this will be prepended if it is not present
+ */
+inline void gcode_M217() {
+  if (strncmp(parser.string_arg, "//", 2) == 0) {
+    SERIAL_ECHOLN(parser.string_arg);
+  } 
+  else {
+    SERIAL_ECHOLNPAIR("// ", parser.string_arg);
+  } 
+}
```

Comment:
OK.  

I'll double check that adding `echo:` doesn't break OctoPrint's parsing of action commands.  My objective isn't necessarily to get something echoed in the host's console, it's to be able to get the host to do something when it receives some output from the machine.